README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# About Me
+
+## Bio
+
+![*This is me*](Picture_hw1.jpg)
+
+### *Introduction*
```

Comment:
Headers automatically add emphasis (but not italics) so good job using that here.